src/reference/asciidoc/ip.adoc

```diff
@@ -1368,7 +1369,7 @@ The framework includes acknowledgment information in the data packet.
 | CONNECTION_ID
 | A unique identifier for a TCP connection; set by the framework for inbound messages; when sending to a server-side inbound channel adapter, or replying to an inbound gateway, this header is required so the endpoint can determine which connection to send the message to.
 | ip_actualConnectionId
-| ACTUAL_CONNECTION_ID
+| ACTUAL_ CONNECTION_ID
```

Comment:
It has to break to avoid overflowing the table cell - it renders ok and I think it's still understandable.